components/renku_data_services/session/blueprints.py

```diff
@@ -92,12 +111,7 @@ def get_all(self) -> BlueprintFactoryResponse:
         @authenticate(self.authenticator)
         async def _get_all(_: Request, user: base_models.APIUser) -> JSONResponse:
             launchers = await self.session_repo.get_launchers(user=user)
-            return json(
-                [
-                    apispec.SessionLauncher.model_validate(item).model_dump(exclude_none=True, mode="json")
-                    for item in launchers
-                ]
-            )
+            return validated_json(apispec.SessionLaunchersList, [self._dump_launcher(item) for item in launchers])
```

Comment:
question: same here, why is a dump method needed?

Reply:
Done in c2525fdca46b856a1f1b8336e1f95bbe3b5eb584.